kafka/modules/kafka/2.1.1/install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
what's the need for having this installation script duplicated in kafka 2.2.1 and kafka 2.2.1 ?